opentype-shaping-malayalam.md

```diff
@@ -274,11 +274,6 @@ conjunct, the zero-width non-joiner must be used instead. The sequence
 "_Consonant_,Halant,ZWNJ,_Consonant_" should produce the first
 consonant in its standard form, followed by an explicit "Halant".
 
-A secondary usage of the zero-width joiner is to prevent the formation of
-"Reph". An initial "Ra,Halant,ZWJ" sequence should not produce a "Reph",
-where an initial "Ra,Halant" sequence without the zero-width joiner
-otherwise would.
-
```

Comment:
So, a similar bit of "keeping all Indic2 pages similar here" as in the terminology section. We would prefer to describe the ZWJ's meaning the same way on all the pages, but add a note that the Reph-blocking function is irrelevant. I know it can feel like overkill. But people who are hopping back and forth between different pages are a bit less likely to get confused that way. My preference would be to retain the paragraph, but to add below it a "Note: " paragraph -- probably just copying in the same "Note" used in 2.6 (Reph position tagging) or 3.4 (`rphf` GSUB) would do it.

Reply:
Ok.